### DISCUSSION TITLE
My notes are inaccessible, showing up loading but not result

### DISCUSSION THREAD

**Discussion by vagary24** [GitbookIO/community#639] [0. General bugs and questions]
**My notes are inaccessible, showing up loading but not result**
https://app.gitbook.com/o/A54NtbEKDVg1ZYVmY31h/s/TnzabJPNK1va1LrMoKyL/~/changes/1/

**Comment by aikoii** [comment]
Hey @vagary24, I couldn't reproduce the issue you are facing 🤔. Could you review the info on [this page ](https://docs.gitbook.com/help-and-faq/faq/connectivity-issues)and see if anything there might apply in your case?
